src/App.tsx

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    key: 'Nothing was pressed yet',
```

Comment:
```suggestion
    pressedKey: 'Nothing was pressed yet',
```
![image](https://user-images.githubusercontent.com/96409563/217056317-c38bfef4-41de-4a31-a7d8-b21bd1af7ecc.png)